README.md

```diff
@@ -185,6 +185,9 @@ player.chromecast(); // initializes the Chromecast plugin
      If nothing is returned or if this option is not defined, no custom data will be sent.
      This option is intended to be used with a [custom receiver][custom-receiver] application
      to extend its default capabilities.
+   * **`chromecast.modifyLoadRequestFn`** - a function that this plugin calls before doing
```

Comment:
Thank you @kontrollanten . You can resolve the markdown lint issue by adding link references at the bottom of the README.

```md
[chromecast-load-media]: https://developers.google.com/cast/docs/reference/web_sender/cast.framework.CastSession#loadMedia
[chromecast-load-request]: https://developers.google.com/cast/docs/reference/web_sender/chrome.cast.media.LoadRequest
```

Then referencing them in the markdown:
```md
   * **`chromecast.modifyLoadRequestFn`** - a function that this plugin calls before doing
     the request to [load media][chromecast-load-media]. The function gets called with the
     [LoadRequest][chromecast-load-request] object as argument and expects it in return.
```

Reply:
It's not working since the URL is 97 characters, and max line length is 90. Is it possible to ignore some lines? Otherwise I'll use some URL shortener.

Reply:
If you run `npm run markdownlint` will any errors occur?

The linter should respect long length URLs if they are link references. See https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md013---line-length

Reply:
Thanks, it worked!